.npmrc

```diff
@@ -1,2 +1,2 @@
 registry=https://registry.npmjs.org/
-engine-strict=true
+engine-strict=false
```

Comment:
looks like the comment got dropped, but FYI this is because the testing tools (ts-jest) say 16+ only. not a big deal--this doesnt ship _with_ the package, and all the testing & dev passes fine.